src/bsoncxx/test/CMakeLists.txt

```diff
@@ -12,24 +12,26 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-include_directories(
-    ${MONGO_CXX_DRIVER_SOURCE_DIR}/src/third_party/catch/include
-)
+if (BUILD_TESTS STREQUAL "ON")
```

Comment:
Does the existing `ENABLE_TESTS` option already cover this? The `add_subdirectory(test)` is guarded by the check on that variable.